pkg/cspc/algorithm/build_csp.go

```diff
@@ -58,6 +58,10 @@ func (ac *Config) GetCSPISpec() (*cstor.CStorPoolInstance, error) {
 		poolSpec.PoolConfig.PriorityClassName = ac.CSPC.Spec.DefaultPriorityClassName
 	}
 
+	if poolSpec.PoolConfig.ROThresholdLimit == 0 {
+		poolSpec.PoolConfig.ROThresholdLimit = 85
+	}
+
```

Comment:
If we convert `ROThresholdLimit` to a pointer -- it would then become non-ambiguous to say intentional zero value of a field.

Reply:
Done